### PR TITLE
HL-274

### PR DESCRIPTION
**Pull request by brettvniekerk**
[Link to ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82?selectedIssue=HL-274)

- Refactors hooks and moves them into descriptive folders
- Tested on local 
- Includes lint

### CODE REVIEW

**Review by bitcoinbrisbane — DISMISSED** [review]
Ive pulled and tested locally.  LGTM

**Review by OptimusOpus — APPROVED** [review]
LGTM

File: src/components/Modals/DepositVaultModal.tsx [hunk 1]
```diff
@@ -1,36 +1,60 @@
...
-  const { balance, refetch: refetchUserBalance } = useUserBalance(
-    vault.asset.address,
-    signer
-  );
   const { deposit } = useVaultContract();
+  const { getBalance, getDecimals } = useERC20Contract();
+  const { shouldRefetch, refetch: refetchUserBalance } = useRefetch();
+
+  useEffect(() => {
+    if (!signer || !config) return;
```
Comment (JasminFarhi) [inline src/components/Modals/DepositVaultModal.tsx:39]: do we need the config?

File: src/components/Modals/PlaceBetModal.tsx [hunk 2]
```diff
@@ -56,14 +55,36 @@ export const PlaceBetModal: React.FC<Props> = ({
     };
   }, [runner]);
 
+  useEffect(() => {
+    if (!selectedMarket || !signer || !config) return;
```
Comment (JasminFarhi) [inline src/components/Modals/PlaceBetModal.tsx:59]: same here?

File: src/components/Modals/DepositVaultModal.tsx (hunk 1 above)
Comment (brettvniekerk) [reply src/components/Modals/DepositVaultModal.tsx]: caught it already - good comment tho

**Review by bitcoinbrisbane — APPROVED** [review]

**Review by JasminFarhi — APPROVED** [review]
LGTM

File: src/components/Modals/PlaceBetModal.tsx (hunk 2 above)
Comment (brettvniekerk) [reply src/components/Modals/PlaceBetModal.tsx:59]: no its needed here

**Pull request merged**